### DISCUSSION TITLE
Maintaining Daily Commits for an Active and Intuitive GitHub Profile

### DISCUSSION THREAD

**Discussion by lucas-0liveira** [dendrovis/github#48] [Q&A] [answered]
How can I ensure there are 'commits' on my GitHub profile every day? I'd like to keep it active and user-friendly, I've seen profiles with a lot of commits.

**Comment by dendrovis** [comment, marked as answer]
0.0 didn't expect anyone to put a qa here unless for the sake for maintaining active GitHub activity. This repo is just merely for my own testing purpose fyi. To answer your question

1. active github
  - There are many options to keep it active be it cheat way such as cron jobs to non-cheat way finding things to contribute. But in general if you are looking to impress someone, i think this may not be the right approach

2. user friendly github
  - This is something limited base on how github team allow github user to control their account so it may not be your best interest


Why not think of another approach to satisfy point 2, use your skill to the test such as building your own portal and use that as your branding, you can design as user-friendly and intuitive as you like. 

Hope this answer your question although it has been overdue i believe hehe

Keep up in whatever things you are doing and have a great day! 

Cheers,
Dendrovis Contributor
